app/battle.py

```diff
@@ -0,0 +1,39 @@
+from app.knight import Knight
+
+
+class Battle:
+    battle_result = {}
```

Comment:
Consider initializing `battle_result` as an instance variable rather than a class variable if you intend for each `Battle` instance to have its own result. If it's meant to be shared across all instances, then it's fine as is.